packages/main/src/themes/sap_horizon/Carousel-parameters.css

```diff
@@ -0,0 +1,3 @@
+:root {
```

Comment:
You should import the base parameters at the top and then have the parameter defined below to override any of the vars in base:
```css
@import "../base/Carousel-parameters.css"
```

then in `packages/main/src/themes/sap_horizon/parameters-bundle.css` update the import to point to this file rather than the base file

Currently this code works because the base parameters are still being imported in the `parameters-bundle.css`.

Please do this for all uses of `Carousel-parameters.css`.